pkg/network/protocols/http/testutil/testdata/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+name: python
+services:
+  python:
+    image: python:3-alpine
```

Comment:
you need to open a PR (like [this](https://github.com/DataDog/ami-builder/pull/193/)) to add the image for the CI, otherwise the test might failed with `error 429` from dockerhub

Reply:
I don't see any test failing, are you sure this is required?